docker/generic/start_proxy.py

```diff
@@ -548,6 +548,13 @@ def make_argparser():
         Otherwise use ignored_query_parameters. Defaults to false.
         ''')
 
+    parser.add_argument(
+        '--connection_buffer_limit_bytes', action=None,
+        help='''
+        Configure the maximum amount of data that is buffered for each 
+        request/response body. This should be increased for large bodies used 
+        by gRPC transcoding. Default is 1 MB.
```

Comment:
Do we need to mention for gRPC?  what happen to HTTP?

Reply:
It's very unclear to me how this buffering works in Envoy. When I set this flag to a small value and ran all our integration tests, only the gRPC transcoding tests failed. All gRPC and HTTP pass-through tests worked just fine. Hence it seems this will only be a problem for gRPC transcoding, which is why I explicitly mention it here.

Reply:
In theory, it should apply to http too. I prefer not to mention grpc here.

Reply:
Done. What do you think about documenting the default as I did here?